UnityProject/Assets/Scripts/HealthV2/Living/LivingHealthMasterBase.cs

```diff
@@ -1319,13 +1319,14 @@ private void AddToSurface(ReagentMix Chemicals, ReagentMix mix)
 		[Server]
 		public virtual void OnGib()
 		{
-			_ = SoundManager.PlayAtPosition(CommonSounds.Instance.Slip, gameObject.transform.position,
+			SoundManager.PlayAtPosition(CommonSounds.Instance.Slip, gameObject.transform.position,
 				gameObject); //TODO: replace with gibbing noise
-			CirculatorySystem.Bleed(GetTotalBlood());
+			CirculatorySystem.OrNull()?.Bleed(GetTotalBlood());
 			Death();
 			for (int i = BodyPartList.Count - 1; i >= 0; i--)
 			{
-				BodyPartList[i].TryRemoveFromBody(true);
+				if (BodyPartList[i].BodyPartType == BodyPartType.Chest) continue;
```

Comment:
Is this a fix for the freeze or Just making it so you have a torso still?

Reply:
making sure that the torso is still there